README.md

```diff
@@ -122,16 +140,26 @@ Note that as of November 6, 2023, OpenShift Serverless Operator is based on RHEL
     > - `ARGOCD_USERNAME`: Default value is set to `admin`.
     > - `ARGOCD_PASSWORD`: This value is dynamically obtained based on the first ArgoCD instance available.
 
-3.  Install the orchestrator Helm chart:
+9.  Create the secret in the 'rhdh-operator' namespace with the keys listed earlier and the GITHUB_TOKEN:
+
+   ```console
+   oc create secret generic backstage-backend-auth-secret \
```

Comment:
can we create a script for that - we can change https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/hack/setenv.sh for that purpose since it is no longer useful.

Reply:
Change/rename it sounds good. The part that gathers the values from the user is still useful.

Reply:
The script is still being used to initialize the `WORKFLOW_NAMESPACE` as well as the `ARGOCD_NAMESPACE`:
https://github.com/parodos-dev/orchestrator-helm-chart/blob/2dc26fb40de731e03ece2d1ff041d73b134f579b/README.md?plain=1#L116
https://github.com/parodos-dev/orchestrator-helm-chart/blob/2dc26fb40de731e03ece2d1ff041d73b134f579b/README.md?plain=1#L120

which are not used in the secret. 

I would suggest to keep the script and change it in a way that it still exposes these 2 environment variables and also creates the secret.

Otherwise these 2 namespaced related variables will need to be provided differently as it is right now. WDYT?

Reply:
@masayag take a look at the last commit:
* These 2 env variables are now captured in the namespace as annotations, so that they can be retrieved by the operator's post script execution.
* The script will now create the secret in the `rhdh-operator` namespace, as well as adding the annotations in the namespace. So no information is lost from running the command.

Reply:
ok so I iterated the idea of the annotations and now they are labels in the target namespaces 
* workflow namespace `rhdh.redhat.com/workflow-namespace` (defaulting to sonataflow-infra in the `setup.sh` script)
```
$> oc describe namespace/sonataflow-infra
Name:         sonataflow-infra
Labels:       kubernetes.io/metadata.name=sonataflow-infra
              rhdh.redhat.com/workflow-namespace=
```
* argoCD namespace: `rhdh.redhat.com/argocd-namespace` (defaulting to orchestrator-gitops in the `setup.sh` script)
```
$> oc describe namespace/orchestrator-gitops
Name:         orchestrator-gitops
Labels:       kubernetes.io/metadata.name=orchestrator-gitops
              rhdh.redhat.com/argocd-namespace=
```

Despite it is not possible in helm to filter out manifests by labels (it is still a cumbersome check of one by one), I feel this is a better approach to what I had initially thought of having these 2 keys as annotations in the `orchestrator` namespace, or initially to have them in a configmap in the orchestrator namespace.

Let me know what you think

Reply:
note that the chart will fail to deploy if neither the labels are found, no value is set in the `values.yaml` and no override is provided in the cli when installing/upgrading the chart.

Reply:
the order of preference is first to get the value in the `values.yaml` and fallback to seek the label when none is found.